src/tags/index.js

```diff
@@ -105,8 +105,10 @@ module.exports = function(fractal){
                             if (name.indexOf('attributes') > -1) {
                                 innerContext[name].merge(value);
                             }
-                            else {
-                                innerContext[name] = value;
+                            // If the parent context key is missing (undefined) or true,
+                            // render the component default value, use passed value otherwise.
+                            else if (typeof value !== 'undefined' && value !== true) {
```

Comment:
`true` is a legitimate value for a context variable (e.g. controlling the state of an element, such as a on/off toggle for a bookmark flag), so I don't think we can include that condition.

```suggestion
                            else if (typeof value !== 'undefined') {
```

Reply:
Yes that's true, good point!